addons/stock/models/stock_inventory.py

```diff
@@ -371,12 +379,32 @@ def create(self, values):
             ('package_id', '=', values.get('package_id')),
             ('prod_lot_id', '=', values.get('prod_lot_id'))])
         res = super(InventoryLine, self).create(values)
-        if existings:
-            raise UserError(_("You cannot have two inventory adjustements in state 'in Progess' with the same product"
-                              "(%s), same location(%s), same package, same owner and same lot. Please first validate"
-                              "the first inventory adjustement with this product before creating another one.") % (res.product_id.name, res.location_id.name))
+        if not self.env.context.get('no_validate_from_line', False) and existings:
+            raise UserError(_("You cannot have two inventory adjustments in progress for one product ( %s ), "
+                              "in the same location, for the same package, the same owner and the same lot number. "
+                              "Please validate or cancel the first inventory adjustment before creating a new one.") % (res.product_id.name))
         return res
 
+    @api.multi
+    def check_multi_line(self):
+        res = False
+        for record in self:
+            values = record.read()[0]
+            existings = self.search([
+                ('product_id', '=', values.get('product_id') and values.get('product_id')[0] or False),
+                ('inventory_id.state', '=', 'confirm'),
+                ('location_id', '=', values.get('location_id') and values.get('location_id')[0] or False),
+                ('partner_id', '=', values.get('partner_id') and values.get('partner_id')[0] or False),
+                ('package_id', '=', values.get('package_id') and values.get('package_id')[0] or False),
+                ('prod_lot_id', '=', values.get('prod_lot_id') and values.get('prod_lot_id')[0] or False)])
```

Comment:
Why read the record and use that `values.get` mess instead of using the record directly?